crates/sui/src/keytool.rs

```diff
@@ -69,9 +67,12 @@ pub enum KeyToolCommand {
         /// The alias must start with a letter and can contain only letters, digits, hyphens (-), or underscores (_).
         new_alias: Option<String>,
     },
-    /// Convert private key from wallet format (hex of 32 byte private key) to sui.keystore format
-    /// (base64 of 33 byte flag || private key) or vice versa.
+    /// Convert private key from legacy format (hex of 32 byte private
+    /// key) to new format (bech32 encoded 33 byte flag || private key
```

Comment:
Be consistent in naming. Below we use `Bech32` whereas here it is `bech32`.
```suggestion
    /// key) to new format (Bech32 encoded 33 byte flag || private key
```